### PR TITLE
Remove acra-connector from timescale example

### PR DESCRIPTION
**Pull request by Lagovas**
* remove acra-connector usage
* update docker images to the newest versions
* configure timescale to use TLS certs
* update pgadmin configuration with pre-configured files

### CODE REVIEW

**Review by shadinua — APPROVED** [review]

File: timescaledb/scripts/entry.sh [hunk 1]
```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
-export POSTGRESQL_CONNSTR="postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}"\
-"@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}?sslmode=disable"
+export PGSSLCERT=/scripts/acra-client.crt
+export PGSSLKEY=/scripts/acra-client.key
+export PGSSLROOTCERT=/scripts/root.crt
+export PGSSLMODE=require
```
Comment (ZhmakaAS) [inline timescaledb/scripts/entry.sh:6]: Why not `verify-full`?

File: _common/pgadmin/servers.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "Servers": {
+    "1": {
+      "Name": "TimescaleDB",
+      "Host": "postgresql",
+      "Group": "Acra - TimescaleDB example",
```
Comment (ZhmakaAS) [inline _common/pgadmin/servers.json:6]: Hm, is it only related to `TimescaleDB`, if I am not mistaken, pgAdmin image is using almost in all demos we have

File: timescaledb/docker-compose.timescaledb.yml [hunk 3]
```diff
@@ -2,88 +2,60 @@ version: "3"
...
             - "5432:5432"
         networks:
-            - server-timescaledb
-            - postgresqlweb-timescaledb
-            - world
-
+            server-timescaledb:
+              # re-use ssl/pgsql/postgresql.crt with dns name 'postgresql'
+              aliases:
+                - "postgresql"
```
Comment (ZhmakaAS) [inline timescaledb/docker-compose.timescaledb.yml:46]: Cool, I didnt know about such approach:)

File: timescaledb/docker-compose.timescaledb.yml [hunk 4]
```diff
@@ -93,89 +65,62 @@ services:
         # - 'connector-server' - for AcraServer and AcraConnector interconnection
         networks:
             - server-timescaledb
-            - connector-server
+            - metricsource-server
+            - grafana-server
+            - prometheus-server
+        ports:
+            - "9393:9393"
```
Comment (ZhmakaAS) [inline timescaledb/docker-compose.timescaledb.yml:72]: Do we need to open Acra ports to the host? I thought it is only for internal usage

File: _common/pgadmin/servers.json (hunk 2 above)
Comment (Lagovas) [reply _common/pgadmin/servers.json:6]: yeah, you are right. forgot about that. will change to some common `Acra - Postgresql example`.

File: timescaledb/docker-compose.timescaledb.yml (hunk 4 above)
Comment (Lagovas) [reply timescaledb/docker-compose.timescaledb.yml:72]: previously we exported acra-connector's ports to allow users connect via `psql` from host machine. there are same reason, to allow connect to acra not only by pgadmin, but by any other tool that can work with db: psql, DataGrid, etc

File: timescaledb/scripts/entry.sh (hunk 1 above)
Comment (Lagovas) [reply timescaledb/scripts/entry.sh:6]: agree, will change

File: _common/pgadmin/pgadmin.dockerfile [hunk 5]
```diff
@@ -1,9 +0,0 @@
-FROM dpage/pgadmin4:4.29
```
[the remark this answers is not in the document]
Comment (Lagovas) [reply _common/pgadmin/pgadmin.dockerfile:1]: yeah, you are right. I forgot about that and it has used in other examples... will check and update them too

File: timescaledb/docker-compose.timescaledb.yml (hunk 4 above)
Comment (ZhmakaAS) [reply timescaledb/docker-compose.timescaledb.yml:72]: Got it, make sense

**Comment by Lagovas** [comment]
i'm bored with updating run.sh to run and check all other examples locally, so I refactored a bit run.sh to allow run it locally without downloading repo and without local patching run.sh file.
* now if you call `./run.sh django` from already cloned repository, then it just run locally this example
* if you run it with `curl <path> | bash -s -- django` then it will work too
* additionally I added avoiding failures of cloning related repos (acra repository for python example) if it is already exists (in case of local development when you didn't clean it before next run)
* updated alpine version in other dockerfiles and added required `py3-pip` package for new version
* removed `acra-engineering-demo` prefix for `COPY` commands to allow use it from local folder, not only cloned with `curl`, and changed context from `../../` to `../` to use relative from repo root path.
* also applied all suggestions:
* - change `require` to `verify-full` sslmode for timescale
* - renamed setting group for timescale to more common variant

so, @shadinua , @ZhmakaAS look one more time please due to much more changes

**Review by shadinua — APPROVED** [review]

File: django-transparent/django.dockerfile [hunk 6]
```diff
@@ -1,7 +1,4 @@
-FROM alpine:3.10
```
Comment (ZhmakaAS) [inline django-transparent/django.dockerfile:1]: Hm, there was some explanation about newest alpine version and why it was not working with alpine:3.15.0. Have you checked if it is actually working demo with such base image version?

Comment (Lagovas) [reply django-transparent/django.dockerfile:1]: it has worked, I checked. But anyway, updated hash commit for django project to version with newer psycopg2 version that fixes issues related to python3.8

**Review by ZhmakaAS — APPROVED** [review]
Lgtm, thanks

**Pull request merged**